.gitignore

```diff
@@ -11,3 +11,5 @@
 /web/bundles/
 /app/SymfonyRequirements.php
 /app/check.php
+
+/.idea/
```

Comment:
this is specific to your dev environment and should be in your system git ignore, not in the project

Reply:
there we go...